x-pack/plugins/synthetics/server/synthetics_service/project_monitor/normalizers/common_fields.ts

```diff
@@ -143,6 +143,16 @@ export const getMultipleUrlsOrHostsError = (
   )} per monitor. You monitor was not created or updated.`,
 });
 
+export const getNoUrlsOrHostsError = (
+  monitor: ProjectMonitor,
+  key: 'host' | 'url',
+  version: string
+) => ({
+  id: monitor.id,
+  reason: 'Unsupported Heartbeat option',
```

Comment:
```suggestion
  reason: 'Invalid Heartbeat configuration',
```

Reply:
I'm sorry. I really thought we had done i18n for these errors. I'm realizing now I'm mistaken.

Reply:
Also, I'm happy to involve @florent-leborgne in this now if that's alright. 

Florent, we have these errors that are reported back to our synthetics cli from the Kibana server.

This one in particular happens when a user omits a required field.

Reply:
Seeing the screenshot in the PR description, your suggested change lgtm @dominiqueclarke. Thanks for pinging me. If there are more strings to review that I cannot find/edit in this PR, let me know and I'll be happy to take a look.